grafana/map.jinja

```diff
@@ -7,6 +7,29 @@ Debian:
   bind:
     address: 0.0.0.0
     port: 3000
+  session:
+    engine: file
+  auth:
+    engine: application
+  admin:
+    user: admin
+    password: admin
+  allow_sign_up: False
+  allow_org_create: False
+  auto_assign_role: Viewer
```

Comment:
Out of curiosity, why did you add the default values to this file instead of metadata/service/server.single.yml?
I see that depending on the formulas, default values are added to the service metadata, map.jinja or both...